features/admin.users.v1/components/wizard/steps/add-user-basic.tsx

```diff
@@ -126,13 +174,61 @@ export const AddUserUpdated: React.FunctionComponent<AddUserProps> = (
     const [ isFirstNameRequired, setFirstNameRequired ] = useState<boolean>(true);
     const [ isLastNameRequired, setLastNameRequired ] = useState<boolean>(true);
     const [ isEmailRequired, setEmailRequired ] = useState<boolean>(false);
+    const [ profileSchema, setProfileSchema ] = useState<ProfileSchemaInterface[]>();
+    const [ multiValuedInputFieldValue, setMultiValuedInputFieldValue ] = useState<Record<string, string>>({});
+    const [ multiValuedAttributeValues, setMultiValuedAttributeValues ] =
+        useState<Record<string, string[]>>({});
+    const [ primaryValues, setPrimaryValues ] = useState<Record<string, string>>({}); // For multi-valued attributes.
+    const [ isMultiValuedItemInvalid, setIsMultiValuedItemInvalid ] =  useState<Record<string, boolean>>({});
+    const [ simpleMultiValuedExtendedProfileSchema, setSimpleMultiValuedExtendedProfileSchema ]
+        = useState<ProfileSchemaInterface[]>();
+    const [ countryList, setCountryList ] = useState<DropdownItemProps[]>([]);
```

Comment:
Do we need a state variable for this? Since the country list is loaded in the initial rendering and unchanged thereafter, we can avoid using a state variable.

Reply:
Fixed with 5cbff8089174971c91a1c1aabd4c1a11270bdb83